spec/magma_attribute_spec.rb

```diff
@@ -177,5 +177,14 @@
 
       expect(attribute.validation_object.validate("A")).to eq(true)
     end
+
+    it "contains a ValidationOjbect" do
+      model = double("model", project_name: :project, model_name: :model)
+      attribute = Magma::Attribute.new("name", model, {
+        validation: { type: "Regexp", value: /^[a-zA-Z]{1}$/ }
+      })
+
+      expect(attribute.json_template[:validation_object]).to be_a(Magma::RegexpValidationObject)
```

Comment:
I think `json_template[:validation_object]` should return JSON so Timur can consume it. One option could be to define `to_json` on `Magma::ValidationObject`. Then, we could call `validation_object.to_json` in `#json_template`.